src/main/java/com/example/model/Book.java

```diff
@@ -8,9 +8,13 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import java.util.Objects;
+import lombok.Getter;
+import lombok.Setter;
 
 @Entity
 @Table(name = "books")
+@Getter
+@Setter
```

Comment:
You can use other lombok annotations and replace overridden methods but use it correctly ;)